flavors/standard-EXASOL-all-java-11/flavor_base/base_test_build_run/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM {{language_deps}}
+
+RUN mkdir /conf /buckets
+
+COPY --from={{base_test_deps}} /usr /usr
+RUN true # workaround for https://github.com/moby/moby/issues/37965
+
+COPY --from={{base_test_deps}} /lib /lib
+RUN true # workaround for https://github.com/moby/moby/issues/37965
+
+COPY --from={{base_test_deps}} /bin /bin
+RUN true # workaround for https://github.com/moby/moby/issues/37965
+
+COPY --from={{base_test_deps}} /opt /opt
+RUN true # workaround for https://github.com/moby/moby/issues/37965
+
+COPY --from={{base_test_deps}} /etc /etc
+RUN true # workaround for https://github.com/moby/moby/issues/37965
+
+COPY --from={{base_test_deps}} /env /env
+RUN true # workaround for https://github.com/moby/moby/issues/37965
+
+
+RUN ldconfig
+
+RUN mkdir /exaudfclient /exaudf
+COPY /exaudfclient /exaudfclient
+
+WORKDIR /exaudfclient/
+RUN ["/bin/bash", "-c", "source /env && bash build.sh --config no-tty -c dbg --config java --config test-binaries"]
+RUN cp -r -L bazel-bin/* /exaudf && rm -r /exaudf/external
+RUN rm -r /root/.cache/bazel #Delete bazel cache as it contains java rules with special characters which will cause a problem for Bucketfs
+WORKDIR /exaudfclient/base
+RUN ./test_udfclient.sh /exaudf/exaudfclient
+RUN ./test_udfclient.sh /exaudf/exaudfclient_static
+
+WORKDIR /
+RUN mkdir /exasol_emulator
+COPY emulator/ /exasol_emulator
+COPY /exaudfclient/base/exaudflib/zmqcontainer.proto /exasol_emulator
+RUN cd /exasol_emulator && protoc zmqcontainer.proto --python_out=.
```

Comment:
When I see this, this basically repeated for all flavors, I have the feeling, we should extract that into a script at some point

Reply:
Created https://github.com/exasol/script-languages-release/issues/1083